tests/foreman/maintain/test_backup_restore.py

```diff
@@ -30,8 +30,7 @@
 
 
 BASIC_FILES = {"config_files.tar.gz", ".config.snar", "metadata.yml"}
-OFFLINE_FILES = {"pgsql_data.tar.gz", ".postgres.snar"} | BASIC_FILES
-ONLINE_SAT_FILES = {"candlepin.dump", "foreman.dump", "pulpcore.dump"} | BASIC_FILES
+ONLINE_SAT_FILES = OFFLINE_FILES = {"candlepin.dump", "foreman.dump", "pulpcore.dump"} | BASIC_FILES
 ONLINE_CAPS_FILES = {"pulpcore.dump"} | BASIC_FILES
```

Comment:
Hmm, I think we need to introduce `OFFLINE_CAPS_FILES` now -- or rather use `ONLINE_CAPS_FILES` always for capsules (as those won't have `foreman.dump` and `candlepin.dump`).

Probably just rename it to `CAPS_FILES` then.
And then we can also rename `ONLINE_SAT_FILES` and `OFFLINE_FILES` to `SAT_FILES`.

Reply:
Updated the PR